### ISSUE TITLE
Create getting started screencast

### ISSUE THREAD

**Issue by simitt** [elastic/apm-mutating-webhook#54]
Create a short screencast, showing how to get started. 

Rough layout:
* already have a k8s cluster with a java and nodejs app up & running (might be reusing opbeans)
* already have a Elastic Stack with APM Server running (referencing Elastic Cloud and/or ECK)
* show minimal steps how to instrument existing applications with the k8s webhook and show data showing up in APM UI. 

Goal:
* Show how to get started with the webhook with a minimal set of required steps (basic configurations of server and secret etc.), withouth going into any details on how to build the webhook etc. 
* No prior knowledge about webhooks should be required. 
* The screencast should be ready to be made available publicly.

**Comment by marclop** [comment]
Before I record the screencast, would we want to wait for the helm chart to be renamed? or is that orthogonal?

**Comment by felixbarny** [comment]
Yes, I think we should do that first, so that the screencast isn't outdated on arrival.

**Comment by felixbarny** [comment]
Specifically, let's try to resolve these naming discussions before recording the screencast:
- https://github.com/elastic/apm-mutating-webhook/issues/52
- https://github.com/elastic/apm-mutating-webhook/issues/28
